doc/tutorials/kmeans/kmeans.txt

```diff
@@ -140,7 +140,7 @@ last iteration when the cluster was not empty.
 $ mlpack_kmeans -c 5 -i dataset.csv -v -e -o assignments.csv -C centroids.csv
 @endcode
 
-@subsection cli_ex3_kmtut Killing empty clusters
```

Comment:
Likewise, just trying to avoid duplications. "tut"->"tu" probably isn't a good solution.